### PR TITLE
conversion to markdown

### PR DESCRIPTION
**Pull request by jantley-ibm**

### CODE REVIEW

**Comment by jantley-ibm** [comment]
not crazy about the inline files; suggestions?
couldn't bring myself to delete the multipane asciidoc yet

File: guide-app-monitoring-ocp4.2.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+For application monitoring on RHOCP, you need to set up your own Prometheus and Grafana deployments. Both Prometheus and Grafana can be set up via Operator Lifecycle Manager (OLM).
+
+## Prerequisites
+
+Prior to deploying Prometheus, ensure that there is a running application that has a service endpoint for outputting metrics in Prometheus format.
+
+It is assumed such a running application has been deployed to the RHOCP cluster inside a project/namespace called `myapp`, and that the Prometheus metrics endpoint is exposed on path `/metrics`.
+
+Git clone this repo to get going right away:
```
Comment (SueChaplain) [inline guide-app-monitoring-ocp4.2.md]: @jantley-ibm - would we need to include this if we are now linking to files?

Comment (jantley-ibm) [reply guide-app-monitoring-ocp4.2.md]: @SueChaplain I'm OK with removing it; will make this change

**Comment by jantley-ibm** [comment]
@leemelim did the images that appear in this guide come from open source?

**Comment by halim-lee** [comment]
> @leemelim did the images that appear in this guide come from open source?

prometheusOperator.png was from https://kabanero.io/guides/app-monitoring. I am not sure where Frank(@fwji) got the image from.

prometheus_endpoints.png and template_grafana_dashboard.png are my screenshots.

**Review by halim-lee — APPROVED** [review]

**Comment by jantley-ibm** [comment]
@fwji could you indicate the original source of the prometheusOperator.png image?

**Comment by alohr51** [comment]
@jantley-ibm so this Guide will be difficult to move to markdown. It's using the special 2 pane Guide style https://kabanero.io/guides/app-monitoring-ocp4.2 which is asciidoc only. We would have to redo how the code is shown and it won't look as nice.

 I would hold off on this one for now.

**Comment by fwji** [comment]
@jantley-ibm the figure is from coreos.com. Coreos is the original creator of operator and Prometheus operator, it was later acquired by red hat.

**Comment by jantley-ibm** [comment]
Decided not to convert this one to markdown in Kabanero. Closing this PR without merging.

**Pull request closed without merging**